packages/experimental/RadioGroup/src/Radio/Radio.tsx

```diff
@@ -25,23 +24,20 @@ export const Radio = compose<RadioType>({
   ...stylingSettings,
   slots: {
     root: Pressable,
-    button: View,
+    button: Pressable,
```

Comment:
@gulnazsayed would you mind testing the PR to make sure this change doesn't result in issues with clicking the radio? We had an issue with Checkbox's inner view becoming Pressable, I think. #2466 has details

Reply:
Have verified the behavior on win32 as well! No regressions seen

Reply:
Yep, appears to be working correctly on win32!